src/db/model/stardust/message.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message_stardust::{semantic::ConflictReason, Message, MessageId};
+use mongodb::bson::doc;
+use serde::{Deserialize, Serialize};
+
+use crate::db::model::{inclusion_state::LedgerInclusionState, Model};
+/// Chronicle Message record
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[allow(missing_docs)]
+pub struct MessageRecord {
```

Comment:
What about the rest of the metadata? For example `is_solid`? I think for some data use cases we might be interested in storing even non-solid message (potentially updating them over time).